app.territoiresentransitions.react/src/plans/fiches/share-fiche/fiche-share-info.div.tsx

```diff
@@ -0,0 +1,65 @@
+import { useCollectiviteId } from '@/api/collectivites';
+import { Fiche } from '@/app/app/pages/collectivite/PlansActions/FicheAction/data/use-get-fiche';
+import { FicheResume } from '@/domain/plans/fiches';
+import { Notification } from '@/ui';
+
+type FicheShareInfoDivProps = {
+  fiche: Pick<
+    Fiche,
+    'collectiviteNom' | 'sharedWithCollectivites' | 'collectiviteId'
+  >;
+};
+
+export const getShareIcon = (): string => {
+  return 'share-forward-fill';
+};
```

Comment:
pas besoin d'une fonction ici
```suggestion
export const FICHE_SHARING_ICON = 'share-forward-fill';
```

Reply:
yes